bot/seasons/evergreen/issues.py

```diff
@@ -4,54 +4,71 @@
+        """Command to get issues/pull request from GitHub."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
+        mergeURL = f"https://api.github.com/repos/{user}/{repository}/pulls/{str(number)}/merge"
 
-        if response_code in failed_status:
-            return await ctx.send(failed_status[response_code])
-
-        repo_url = f"https://github.com/{user}/{repository}"
-        issue_embed = discord.Embed(colour=Colours.bright_green)
-        issue_embed.add_field(name="Repository", value=f"[{user}/{repository}]({repo_url})", inline=False)
-        issue_embed.add_field(name="Issue Number", value=f"#{number}", inline=False)
-        issue_embed.add_field(name="Status", value=json_data["state"].title())
-        issue_embed.add_field(name="Link", value=json_data["html_url"], inline=False)
-
-        description = json_data["body"]
-        if len(description) > 1024:
-            placeholder = " [...]"
-            description = f"{description[:1024 - len(placeholder)]}{placeholder}"
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
 
-        issue_embed.add_field(name="Description", value=description, inline=False)
+        if r.status in RESP_VALUE:
+            return await ctx.send(f"[{str(r.status)}] {RESP_VALUE.get(r.status)}")
+        # the original call is made to the issues API endpoint
+        # if a issue or PR exists then there will be something returned
+        # if the word 'issues' is present within the response then we can simply pull the  data we need from the
+        # return data received from the API
+        if "issues" in json_data.get("html_url"):
+            if json_data.get("state") == "open":
+                iconURL = icons.get("issue")
+            else:
+                iconURL = icons.get("issue_closed")
+        # if the word 'issues' is not contained within the returned data and there is no error code then we know that
+        # the requested data is a pr, hence to get the specifics on it we have to call the PR API endpoint allowing us
+        # to get the specific information in relation to the PR that is not provided via the issues endpoint
+        else:
+            async with self.bot.http_session.get(mergeURL) as m:
+                if json_data.get("state") == "open":
+                    iconURL = icons.get("pull-request")
+                # when the status is 204 this means that the state of the PR is merged
+                elif m.status == 204:
+                    iconURL = icons.get("merge")
+                # else by the process of elimination, the pull request has been closed
+                else:
+                    iconURL = icons.get("pull-request-closed")
 
-        await ctx.send(embed=issue_embed)
+        resp = discord.Embed(colour=Colours.bright_green)
+        resp.set_author(
+            name=f"{iconURL} | [{repository}] #{number} {json_data.get('title')}",
```

Comment:
The embed `name` field doesn't support emojis. Rather than making life difficult and parsing the emoji ID to get the link to the image, moving the information into the body of the embed works fine:

```py
        resp = discord.Embed(
            colour=Colours.bright_green,
            description=f"{iconURL} [{repository}] #{number} {json_data.get('title')}"
        )
        resp.set_author(name="GitHub", url=json_data.get("html_url"))
        await ctx.send(embed=resp)
```

![image](https://user-images.githubusercontent.com/5323929/66343788-5f5f1880-e91a-11e9-8a8c-d513109d5a45.png)

Reply:
I don't know what to say borther. The goal was to have it on one line as designed with Scragly. That is why the URL's worked in the first place. The use of emojis is creating a probelm here. Let me know your thoughts before I continue.

Reply:
All other changes made.

Reply:
I gave you thoughts & the code to accomplish it, not sure what else there is to clarify.